src/Native/Runtime/inc/eetype.h

```diff
@@ -378,10 +378,7 @@ class EEType
         { return ((m_usFlags & (UInt16)RelatedTypeViaIATFlag) != 0); }
 
     bool IsArray()
-        { return IsParameterizedType() && get_ParameterizedTypeShape() != 0; }
-
-    bool IsPointerType()
-        { return IsParameterizedType() && get_ParameterizedTypeShape() == 0; }
+        { return IsParameterizedType() && get_ParameterizedTypeShape() > 1; }
```

Comment:
It would be nice to at least add `/* ParameterizedTypeShapeConstants.ByRef */` as a comment to make it clear where the magic number came from.